README.md

```diff
@@ -68,6 +68,8 @@ All CI/CD jobs are defined in the [.github/workflows](./.github/workflows) direc
 
 All workflows originating from this repository can be repurposed by other R package Github repositories.
 
+For detailed description and screenshots of workflows please refer to [this document](./workflows.md).
+
 ### Pre-commit
 
 This repository contains an example [pre-commit] configuration.
```

Comment:
Comment to line 77: [pre-commit] is a tool that uses [Git hooks] to identify and resolve simple issues before submission for code review.

Reply:
Updated according to the suggestion.